06week/checkpoint2.js

```diff
@@ -0,0 +1,126 @@
+
+// Given 1000 digits of PI as strings, return an array of the digits as numbers
+
+const nums = strNums.map((number) => {
+  return Number(number)
+});
+
+console.log(nums);
+
+// Find the sum of the even values
+const sumEvens = nums.filter((number) => {
+
+  return ((number)%2===0)
+  
+}).reduce((accumulator, currentValue) => {
+  
+  return (accumulator) + (currentValue);
+    
+});
+  
+console.log(sumEvens);
+
+// Find the index of the first "Austin" value (the value plus its index equals 512)
+
+const atxIdx = nums.forEach((currentValue, index) => {
```

Comment:
Don't use forEach to return a value because the return of forEach is always 'undefined'.

Try using the find method here to solve this problem